iFixit/react-commerce
# Redirects: Export legacy filter slugs for PHP

We want to be able to use these redirects in PHP too, so let's pull these into a json file so that PHP can reach into the submodule and easily read them.

## QA
Make sure item type and tool category redirects aren't broken. See the QA from https://github.com/iFixit/react-commerce/pull/524

CC @sterlinghirsh 

Connects #599
